README.md

```diff
@@ -24,6 +24,7 @@
 * [IRC - #nixos on freenode.net](irc://irc.freenode.net/#nixos)
 * [NixOS Weekly](https://weekly.nixos.org/)
 * [Community-maintained wiki](https://nixos.wiki/)
+* [Community-maintained chat alternatives](https://nixos.wiki/wiki/Get_In_Touch#Chat) (Discord, matrix, telegram, other IRC channels, etc.)
```

Comment:
I know you are focused on the chat aspect, but it seems like just linking to the top level makes sense like:

```suggestion
* [Community-maintained list of ways to get in touch](https://nixos.wiki/wiki/Get_In_Touch)
```